imap_processing/tests/glows/test_glows_l1a_data.py

```diff
@@ -22,6 +25,22 @@ def decom_test_data():
     return data_packet_list
 
 
+@pytest.fixture()
+def l1a_test_data(decom_test_data):
+    hist_l1a = []
+
+    for hist in decom_test_data[0]:
+        hist_l1a.append(HistogramL1A(hist))
+
+    de_l1a_dict = process_de_l0(decom_test_data[1])
+
+    de_l1a = []
+    for value in de_l1a_dict.values():
+        de_l1a += value
```

Comment:
Minor nit to make this similar to your other test code block
```suggestion
    hist_l1a = [HistogramL1A(hist) for hist in decom_test_data[0]]

    de_l1a_dict = process_de_l0(decom_test_data[1])

    de_l1a = [value for value in de_l1a_dict.values()]
```